docs/source/delta-spark-connect.md

```diff
@@ -0,0 +1,132 @@
+---
+description: Learn about Delta Connect - Spark Connect Support in Delta.
+---
+
+# TODO update this for the 4.0.0 release
```

Comment:
@longvu-db will update this page for 4.0 this just copies over the base src code from the 4.0 preview release